.circleci/config.yml

```diff
@@ -10,22 +10,30 @@ jobs:
       - run:
           name: Check Twitter handles
           command: |
-            gem i twitter --no-post-install-message --no-suggestions --minimal-deps --no-verbose -N -q
-            bash twitter.sh
+            [[ -n "$(git --no-pager diff origin/master..HEAD ../_data | grep "^+[[:space:]]" | grep twitter | cut -c15-)" ]] && { gem i twitter --no-post-install-message --no-suggestions --minimal-deps --no-verbose -N -q; ruby twitter.rb; } || echo "No Twitter handles found. Skipping checks.";
```

Comment:
@phallobst this version built successfully. Does it look good to you?

Reply:
I'd merge the two ` grep`s into one:
```
... | grep "^+[[:space:]]*twitter: " | cut ...
```
That eliminates the risk that "twitter" appears somewhere else in the line without marking the handle itself.

Reply:
Alright I did that change as well as move from git diff to git log.
With git log the outdated stuff won't show up which means no more failing builds just because a fork is outdated 🎉